apps/platforms/win/windows_terminal/windows_terminal.talon

```diff
@@ -18,3 +18,13 @@ kill all:
     insert("y")
     key(enter)
     
+
+(find it | buffy | queer eye)$:
```

Comment:
we probably shouldn't have so many synonyms for one command. One way is preferable to many.

`find it` seems fine to me. Or, elsewhere it looks like we use `hunt` for this functionality, eg. in `text/find_and_replace.talon`.

Also, there's no need to end-anchor this version (`$`), is there? Could be useful to say `find it snake hello world`, for example.